src/engraving/libmscore/masterscore.h

```diff
@@ -151,6 +151,7 @@ class MasterScore : public Score
     void setPlaylistClean() { _playlistDirty = false; }
 
     void setExpandRepeats(bool expandRepeats);
+    bool expandRepeats();
```

Comment:
Please, add const here

Reply:
done. no override?

Reply:
There is no need for this. This method isn't virtual